server/sonar-server/src/main/java/org/sonar/server/usergroups/ws/UsersAction.java

```diff
@@ -37,16 +37,14 @@
 import org.sonar.server.db.DbClient;
 import org.sonar.server.user.UserSession;
 
+import static org.sonar.api.server.ws.WebService.SelectionMode.DESELECTED;
+import static org.sonar.api.server.ws.WebService.SelectionMode.SELECTED;
+
 public class UsersAction implements UserGroupsWsAction {
 
   private static final String PARAM_ID = "id";
-  private static final String PARAM_SELECTED = "selected";
-
-  private static final String SELECTION_ALL = "all";
-  private static final String SELECTION_SELECTED = PARAM_SELECTED;
-  private static final String SELECTION_DESELECTED = "deselected";
 
-  private static final String FIELD_SELECTED = PARAM_SELECTED;
+  private static final String FIELD_SELECTED = "selected";
```

Comment:
I don't think it should use Param.SELECTED: although it holds the same value, the purpose is not the same.